plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -72,33 +158,47 @@ class OptunaConfig:
     # Number of parallel workers
     n_jobs: int = 2
 
-    # Sampling algorithm
-    # Please refer to the reference for further details
-    # https://optuna.readthedocs.io/en/stable/reference/samplers.html
-    sampler: Sampler = Sampler.tpe
-
-    # Random seed of sampler
-    seed: Optional[int] = None
+    search_space: Dict[str, Any] = field(default_factory=dict)
 
-    def __post_init__(self) -> None:
-        if isinstance(self.direction, list) and len(self.direction) > 1:
-            if self.sampler == Sampler.tpe:
-                # The default sampler of multi-objective optimization is NSGAIISampler
-                self.sampler = Sampler.nsgaii
```

Comment:
Okay, so we have lost this logic.

@toshihikoyanase, is it an error to use tpe with multi-objective optimization?
should we fail here? should we issue a warning?
Without introducing a new config group for multi-objective, I don't think we can make the default different (and this seems like an overkill to me).

The example app from @monney is using nsgaii.
We can add a warning or an error if the combination is not supported (but I think it will make more sense if it comes from Optuna itself).

Reply:
@omry good catch. I knowingly dropped this for the reason you outlined. Optuna already triggers an error before the sweep starts if this is set to TPE.
`ValueError: If the study is being used for multi-objective optimization, TPESampler cannot be used.`

Reply:
Yeah, it looks good enough for me.